src/decoder/mod.rs

```diff
@@ -31,6 +31,9 @@ mod vorbis;
 #[cfg(all(feature = "wav", not(feature = "symphonia-wav")))]
 mod wav;
 
+/// Output format of the decoders.
+pub type DecoderFormat = f32;
```

Comment:
"DecoderSampleFormat" or even "DecoderSample", maybe. Just "format" is ambiguous, since this may also refer to a stream format (channels layout, or sample rate). Or someone unfamiliar with the code may imagine that this is format of a decoder itself, somehow.

Reply:
Done in [a1cf45a](https://github.com/RustAudio/rodio/pull/669/commits/a1cf45a02c723043523a017a55386779023e2c70).